check_api/src/main/java/com/google/errorprone/BaseErrorProneJavaCompiler.java

```diff
@@ -31,6 +31,9 @@
 import com.sun.tools.javac.util.JavacMessages;
 import com.sun.tools.javac.util.Log;
 import com.sun.tools.javac.util.Log.WriterKind;
+
+import org.checkerframework.javacutil.AnnotationUtils;
+
```

Comment:
@msridhar IIRC, Google Java style now dictates that all non-static imports must stay together, so I think it's invalid for there to be any blank lines before and after this `org.checkerframework.javacutil.AnnotationUtils` import.

Reply:
@jbduncan fixed (also put it in alphabetical order).  Thanks!

Reply:
You're very welcome @msridhar! :+1: